xml/storage_filesystems.xml

```diff
@@ -474,74 +474,171 @@ Id Path      single   single    single   Unallocated
+       Confirm with <guimenu>Accept</guimenu>.
+      </para>
+      <figure>
+       <title>Setting Quota for a Subvolume</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="yast2_btrfs_quotas_set.png" width="75%"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="yast2_btrfs_quotas_set.png" width="75%"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+      <para>
+       The new size limit will be displayed next to the subvolume name:
+      </para>
+      <figure>
+       <title>List of Subvolumes for a Device</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="yast2_btrfs_quotas_edit.png" width="75%"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="yast2_btrfs_quotas_edit.png" width="75%"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+     </step>
+     <step>
+      <para>
+       Apply changes with <guimenu>Next</guimenu>.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+   <sect3 xml:id="setting-btrfs-quotas-using-cmdline">
+    <title>Setting Btrfs Quotas on the Command Line</title>
+    <para>
+     To set a quota for a subvolume of the root file system on the command
+     line, proceed as follows:
+    </para>
+    <procedure>
+     <step>
+      <para>
+       Enable quota support:
+      </para>
 <screen>&prompt.sudo;btrfs quota enable /</screen>
-    </step>
-    <step>
-     <para>
-      Get a list of subvolumes:
-     </para>
+     </step>
+     <step>
+      <para>
+       Get a list of subvolumes:
+      </para>
 <screen>&prompt.sudo;btrfs subvolume list /</screen>
-     <para>
-      Quotas can only be set for existing subvolumes.
-     </para>
-    </step>
-    <step>
-     <para>
-      Set a quota for one of the subvolumes that was listed in the previous
-      step. A subvolume can either be identified by path (for example
-      <filename>/var/tmp</filename>) or by <literal>0/<replaceable>SUBVOLUME
-      ID</replaceable></literal> (for example <literal>0/272</literal>). The
-      following example sets a quota of 5 GB for <filename>/var/tmp</filename>.
-     </para>
+      <para>
+       Quotas can only be set for existing subvolumes.
+      </para>
+     </step>
+     <step>
+      <para>
+       Set a quota for one of the subvolumes that was listed in the previous
+       step. A subvolume can either be identified by path (for example
+       <filename>/var/tmp</filename>) or by <literal>0/<replaceable>SUBVOLUME
+       ID</replaceable></literal> (for example <literal>0/272</literal>). The
+       following example sets a quota of 5 GB for <filename>/var/tmp</filename>.
+      </para>
 <screen>&prompt.sudo;btrfs qgroup limit 5G /var/tmp</screen>
+      <para>
+       The size can either be specified in bytes (5000000000), kilobytes
```

Comment:
Should the bytes value be 5000000000B?

Reply:
as a basic measure, bytes can live without a unit specifier

Reply:
:+1: